references.bib

```diff
@@ -0,0 +1,66 @@
+@inproceedings{petrillo2010agility,
+  title={Is agility out there?: agile practices in game development},
+  author={Petrillo, Fabio and Pimenta, Marcelo},
+  booktitle={Proceedings of the 28th ACM International Conference on Design of Communication},
+  pages={9--15},
+  year={2010},
+  organization={ACM}
+}
```

Comment:
Check the required fields for `@inproceedings`. It seems that address and month are missing. You will be able to find these in the ACM Digital Library.